islandora.install

```diff
@@ -132,3 +132,18 @@ function islandora_update_7001(&$sandbox) {
   $t = get_t();
   return $t("Islandora database updates complete");
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Add default permissions for viewing inactive or deleted objects.
+ * Match permissions for viewing all objects.
+ */
+function islandora_update_7002(&$sandbox) {
```

Comment:
OK, makes sense @jonathangreen, do you have an example (snippeettt) you could share?.  @rosiel what do you think?